src/main/java/com/google/code/or/OpenReplicator.java

```diff
@@ -310,14 +313,24 @@ public Long millisSinceLastEvent() {
 		return binlogParser.millisSinceLastEvent();
 	}
 
+	public boolean isStopOnEOF() {
+		return stopOnEOF;
+	}
+
+	public void setStopOnEOF(boolean stopOnEOF) {

```

Comment:
is it too paranoid to also set `binlogParser` stopOnEOF (if `binlogParser` defined, i.e. after `start`), or raise?